install.sh

```diff
@@ -319,7 +314,35 @@ unzip() {
 }
 
 get_default_install_dir() {
-    printf %s "${HOME}/.temporalio"
+    printf %s "${HOME}/.temporalio/bin"
+}
+
+get_install_dir() {
+    local _dir
+    _dir="$(get_default_install_dir)"
+    if [ -n "${TEMPORAL_DIR-}" ]; then
+        _dir="$TEMPORAL_DIR"
+    fi
+    printf %s "$_dir"
+}
+
+prompt_for_path() {
+    local _dir="$1"
+
+    cat >"$_dir/env" <<EOL
+#!/bin/sh
+case ":\${PATH}:" in
+    *:"$_dir":*)
+        ;;
+    *)
+        export PATH="$_dir:\$PATH"
+        ;;
+esac
+EOL
+
+    local _source=". $_dir/env"
+    say "If you are using bash, you can add temporal to PATH by appending the following to your user profile (e.g. ~/.bash_profile):"
```

Comment:
```suggestion
    say "If you are using bash, you can add the temporal command to PATH by appending the following to your user profile (e.g. ~/.bash_profile):"
```
To distinguish the command name from our company and product names.